daisi/src/cpps/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ add_subdirectory(common)
 add_subdirectory(message)
 add_subdirectory(negotiation)
 add_subdirectory(to)
+add_subdirectory(third_party)
```

Comment:
We could move it to the root ``third_party`` folder to avoid duplication. But if we want to move the subproject folder out of the SOLA monorepo (maybe someday), this dependency would be missing.
As fsmlite is just a single header, both solutions would be fine for me.

Reply:
I would like to keep it this way. fsmlite.h is not that big.